src/main/java/ulcrs/models/session/Session.java

```diff
@@ -2,53 +2,38 @@
 
 import ulcrs.models.schedule.Schedule;
 
-import java.time.LocalDateTime;
+import java.util.Date;
 
 public class Session {
 
     private String name;
     private Schedule existingSchedule;
-    private LocalDateTime generatedTimestamp;
-    private LocalDateTime lastEditedTimestamp;
+    private Date createdTimestamp;
+    private Date lastEditedTimestamp;
 
-    public void save() {
-        // TODO: implement
-    }
-
-    public boolean saveAs(String filename) {
-        // TODO: implement
-        return false;
+    public Session() {
+        lastEditedTimestamp = createdTimestamp = new Date();
+        name = "session_" + createdTimestamp.getTime() + ".json";
     }
 
     public String getName() {
         return name;
     }
 
-    public void setName(String name) {
```

Comment:
Is this removed to make the name immutable for Sessions? I think that's smart, but if so, I would recommend making a constructor that allows for name input for the session.

Reply:
Added new constructor